transactionsx/losttransactioncleaner.go

```diff
@@ -85,10 +87,8 @@ func (c *LostTransactionCleaner) Run(ctx context.Context) error {
 }
 
 func (c *LostTransactionCleaner) cleanup(ctx context.Context) error {
-	c.logger.Debug("running cleanup", zap.String("uuid", c.uuid),
-		zap.String("bucket", c.atrAgent.BucketName()),
-		zap.String("scope", c.atrScopeName),
-		zap.String("collection", c.atrCollectionName))
+	c.logger.Debug("running cleanup",
+		zaputils.FQDocID("uuid", c.atrAgent.BucketName(), c.atrScopeName, c.atrCollectionName, []byte(c.uuid)))
```

Comment:
I think this should be a uuid field with the UUID, and a FQCollectionName() rather than describing the UUID as being a document name.